### PR TITLE
[5.10.0] Upgrade AdoptOpenJDK 11 version to the latest version - 11.0.9_11-jdk

### PR DESCRIPTION
**Pull request by chirangaalwis**

### CODE REVIEW

File: dockerfiles/centos/is/Dockerfile [hunk 1]
```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK CentOS Docker image
-FROM adoptopenjdk/openjdk11:x86_64-centos-jdk-11.0.8_10
```
Comment (DilanUA) [inline dockerfiles/centos/is/Dockerfile:20]: In the case of this architecture addition x86_64, shall we maintain the same pattern used for the other simultaneous releases?

File: dockerfiles/alpine/is/Dockerfile [hunk 2]
```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk11:jdk-11.0.8_10-alpine
+FROM adoptopenjdk/openjdk11:x86_64-alpine-jdk-11.0.9_11
```
Comment (DilanUA) [inline dockerfiles/alpine/is/Dockerfile:20]: In the case of this architecture addition x86_64, shall we maintain the same pattern used for the other simultaneous releases?

File: dockerfiles/centos/is/Dockerfile (hunk 1 above)
Comment (chirangaalwis) [reply dockerfiles/centos/is/Dockerfile:20]: Yes +1. Let's avoid using the architecture. IIUC, this allows the image to be built according to the platform architecture in the given context.

File: dockerfiles/alpine/is/Dockerfile (hunk 2 above)
Comment (chirangaalwis) [reply dockerfiles/alpine/is/Dockerfile:20]: +1

**Review by DilanUA — APPROVED** [review]

**Pull request merged**